cobalt/browser/debug_console.cc

```diff
@@ -205,7 +205,12 @@ bool DebugConsole::FilterOnScreenKeyboardInputEvent(
 
 void DebugConsole::CycleMode() {
   base::AutoLock lock(mode_mutex_);
-  mode_ = (mode_ + 1) % debug::console::kDebugConsoleModeCount;
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kDisableWebDebugger)) {
+    mode_ = (mode_ + 1) % (debug::console::kDebugConsoleModeHud + 1);
```

Comment:
Sorry, what's the logic for this CycleMode and what's this kDebugConsoleModeHud?

Reply:
When the web debugger is disabled, only the Hud mode of the debug console can work, the other modes rely on the web debugger to function.

Reply:
FYI will update per https://github.com/youtube/cobalt/pull/656#discussion_r1272566939

Reply:
Oh, actually it's not possible, the enum is generated from an IDL.

cobalt/debug/console/debug_console_mode.idl

Reply:
Updated as suggested.